section_faiss_vector_index.go

```diff
@@ -256,7 +256,7 @@ func (v *vectorIndexOpaque) flushVectorSection(vecToDocID map[int64]*roaring.Bit
 }
 
 func removeDeletedVectors(index *faiss.IndexImpl, ids []int64) error {
-	sel, err := faiss.NewIDSelectorBatch(ids)
+	sel, err := faiss.NewIDSelectorArray(ids)
```

Comment:
TODO(maybe for a future PR): should the selector be deleted to free memory, like `sel.Delete()`?

Reply:
Done in the current PR itself.